tox.ini

```diff
@@ -25,6 +25,6 @@ commands = flake8
 basepython = python2.7
 
 [flake8]
-ignore = E251,E128
+ignore = E251,E128,E501
```

Comment:
Nice, but can we increase the `max-line-length` to some reasonable value instead?